### PR TITLE
Improve Windows NVTX support

### PR DESCRIPTION
**Pull request by takagi**
This PR improves NVTX support for Windows environments, following @kmatsuura 's change.

### CODE REVIEW

**Comment by okuta** [comment]
How about using `nvToolsExt64_1.lib` to avoid coping dll file.

**Comment by kmatsuura** [comment]
Add to check existence of nvToolsExt64_1.dll
https://github.com/kmatsuura/chainer/commit/c30521c4cc95f0d2876b1c5a42561ed4dc0d1b65

**Comment by takagi** [comment]
I will follow the fix.

File: cupy/cuda/cupy_cuda.h [hunk 1]
```diff
@@ -24,9 +24,7 @@
 #include <cuda_profiler_api.h>
 #include <cuda_runtime.h>
 #include <curand.h>
-#ifndef _WIN32
```
Comment (okuta) [inline cupy/cuda/cupy_cuda.h]: This modify assumes `os.path.exists(nvtoolsext_path) == True`.
Is it correct?

Comment (takagi) [inline cupy/cuda/cupy_cuda.h]: It does fail if nvToolsExt.h is not found.

Comment (okuta) [inline cupy/cuda/cupy_cuda.h]: OK.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**